terraform/system/gcp/variables.tf

```diff
@@ -15,7 +15,7 @@ variable "name" {
 }
 
 variable "node_type" {
-  default = "n1-standard-1"
+  default = "n1-standard-2"
```

Comment:
`n1-standard-1` has only 1CPU and the `fluentd daemonset` will fail to be scheduled on the `elasticsearch` node for lack of CPU resources.